projects/extension/src/components/ClientError.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+
+interface Props {
+  error?: string
+}
+
+export const ClientError = ({ error }: Props) => {
+  return (
+    <div className="break-words block my-3 mx-6 justify-between pt-1.5 leading-4">
+      Smoldot light client has panicked with error:{" "}
+      <span
+        onClick={() => error && navigator.clipboard.writeText(error)}
+        className="cursor-pointer inline text-red-500 font-bold"
+      >
+        "{error}"
+      </span>
+      . Please copy the error message by clicking it or at the link below, and
```

Comment:
I don't really understand what you're trying to say with "clicking it or at the link below"